### DISCUSSION TITLE
v1.67.1

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#3251] [Announcements]
# Hotfixes
**There is not mobile release for this version**

- Fix issues with the server cannot start
- - - -

As always, please consider supporting the project.

🎉 Cheer! 🎉

## Support

- - - -

And as always, bugs are fixed, and many other improvements also come with this release.

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

## What's Changed
* fix(server): exclude e2e from build by @jrasm91 in https://github.com/immich-app/immich/pull/3250

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.67.0...v1.67.1

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.67.1'>v1.67.1</a>.</em>

**Comment by uninvitedguest** [comment]
Thanks!
